ui/app/Makefile

```diff
@@ -23,19 +23,22 @@ format: elm-env $(ELM_FILES)
 	@$(DOCKER_CMD) elm-format --yes $(ELM_FILES)
 
 test: elm-env
+	@$(DOCKER_CMD) rm -rf elm-stuff/generated-code
 	@$(DOCKER_CMD) elm-format $(ELM_FILES) --validate
 	@$(DOCKER_CMD) elm-test
 
 dev-server:
-	elm-reactor
+	elm reactor
 
 script.js: elm-env format $(ELM_FILES)
 	@echo ">> building script.js"
-	@$(DOCKER_CMD) elm make src/Main.elm --yes --output $(TEMPFILE_JS)
-	@$(DOCKER_CMD) uglifyjs $(TEMPFILE_JS) --compress unused --mangle --output $(@)
+	@$(DOCKER_CMD) rm -rf elm-stuff
+	@$(DOCKER_CMD) elm make src/Main.elm --output $(TEMPFILE_JS)
```

Comment:
we can add the `--optimize` flag, right?

Reply:
108594 bytes on script.js down to 93329 bytes it seems

Reply:
Totally missed this, thanks for noticing!